internal/split/split_service.go

```diff
@@ -0,0 +1,88 @@
+package split
+
+import (
+	"io/ioutil"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/splitio/go-client/v6/splitio/client"
+	"github.com/splitio/go-client/v6/splitio/conf"
+
+	configAPI "github.com/newrelic/newrelic-cli/internal/config/api"
+)
+
+// Split.io client API keys
+// These keys are client-facing and are only used to fetch splits.
+// There is no security risk in exposing these keys, as the only purpose they
+// serve is to retrieve experiments and can not be used to modify anything
+// within our internal Split.io system.
+var (
+	prod      = "8me2vu6v8lhssdkrpenp1uunl9s3bdc8njqp"
```

Comment:
nit: consider something like `prodAPIKey` or `prodKey` for clarity?

Reply:
Let's move those to GH secret, and also support env var to make our testing easier.